src/components/FeatureViews/Accounts/outOfOffice.js

```diff
@@ -1,35 +1,115 @@
 import DialogTitle from '@material-ui/core/DialogTitle';
 import Dialog from '@material-ui/core/Dialog';
 import { DialogActions, DialogContent } from "@material-ui/core";
-import React from 'react';
+import React, { useEffect, useMemo, useState } from "react";
+import Button from "@material-ui/core/Button";
 import { connect } from "react-redux";
+import MenuItem from '@material-ui/core/MenuItem';
+import Select from '@material-ui/core/Select';
+import Grid from "@material-ui/core/Grid";
 
 
-class outOfOffice extends React.Component {
-    constructor(props) {
-        super(props);
-        this.state = {
-        };
-    }
+function OutOfOffice(props) {
+    console.log(props);
+    const setDialog = useState(props.open);
+    const setDate = useState(null);
```

Comment:
[setState](https://reactjs.org/docs/react-component.html#setstate) syntax is like this `const [value, setValue] = useState(initialValue);`